src/app/logic/calculatePoints.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+
+export function calculatePoints(
+  correctAnswersScore,
+  isAnswerCorrect,
+  difficultyLevelsProperty,
+) {
+  const difficultyLevelsProperties =
+    Object.getOwnPropertyNames(DIFFICULTY_LEVELS);
+
+  const isMediumMode =
+    difficultyLevelsProperty === difficultyLevelsProperties[1];
+  const isHardMode = difficultyLevelsProperty === difficultyLevelsProperties[2];
+  if (!isMediumMode && !isHardMode) {
+    throw new Error(
```

Comment:
Remember to remove the error from the end of the code that was throwing the error

Reply:
done!